tests/pkg_openwsn/Makefile

```diff
@@ -43,7 +46,7 @@ USEMODULE += openwsn_6lo_frag
 ## Enable link layer security
 # USEMODULE += openwsn_iee802154e_security
 ## Enable MSF dynamic slot allocation
-#  USEMODULE += openwsn_adaptive_msf
+# USEMODULE += openwsn_adaptive_msf
```

Comment:
Did not look, but I hope style fixes like this are their own commit ;-)

Reply:
Its in a single commit that is adapting the test 0a94afc52272166e5dc54d2f16b5eda66657b42 `tests/pkg_openwsn: adapt to sock api`

Reply:
Ok, but removing this whitespace isn't necessary for the adaptation to the sock API, right? ;-)

Reply:
Nope, point take will move it to its own commit

Reply:
addressed